ipn/prefs_test.go

```diff
@@ -372,30 +388,30 @@ func TestPrefsPretty(t *testing.T) {
 		{
 			Prefs{},
 			"linux",
-			"Prefs{ra=false mesh=false dns=false want=false routes=[] nf=off Persist=nil}",
+			"Prefs{ra=false mesh=false dns=false want=false routes=[] nf=off AutoUpdatePrefs{check=false apply=false} Persist=nil}",
```

Comment:
can we make these less verbose? we abbreviate the other options a lot.

e.g. `up=no`, `up=notify`, `up=yes`?

Reply:
you concerned at all that your chosen `upd` looks a lot like `udp=` which we use elsewhere in the netcheck report lines? your call.